pages/irma-uitleg.md

```diff
@@ -305,6 +305,15 @@ update proces.
 
 ### <a name="architectuur"></a>5. Hoe verschilt IRMA van andere authenticatie systemen als iDIN en idensys?
 
+IRMA is geen authenticatie systeem, maar een identificatie systeem.
```

Comment:
Kan IRMA ook als authenticatiesysteem gebruikt worden?

Reply:
De IRMA app authenticeert de vrijgegeven attributen als zijnde afkomstig van de uitgever, middels de digitale handtekening (echtheidskenmerk) van die uitgever. Dit staat hier op [deze pagina](https://privacybydesign.foundation/irma-uitleg/#motorkap) uitgelegd, en in meer detail [hier](https://irma.app/docs/overview/). In die zin is IRMA dus wel een authenticatiesysteem.

Verder identificeert een IRMA-attribuut niet altijd uniek degene die hem vrijgeeft, denk aan het ouder-dan-18 attribuut. IRMA werkt dus niet altijd identificerend.